scripts/west_commands/runners/x86.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2017 Linaro Limited.
```

Comment:
There is nothing x86-specific about this method of flashing, please remove the x86 naming and call it tftp or something, you are just copying a file to a directory here.